### DISCUSSION TITLE
Import EDB to Siwave and export to Q3D

### DISCUSSION THREAD

**Discussion by AghHossein** [ansys/pyaedt#3033] [Q&A] [answered]
Hi everyone,
Is there a simple command in pyaedt to import EDB to Siwave and export it to Q3D? I would like to use the default export options provided in SIwave for the export. I would appreciate your thoughts.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @AghHossein ,

Please could you use for future questions the Ansys developer portal? Then the community will be able to access to the solutions: https://discuss.ansys.com/

I think this is the method you are looking for:
https://aedt.docs.pyansys.com/version/stable/EDBAPI/_autosummary/pyaedt.edb.Edb.export_q3d.html#export-q3d
